### ISSUE TITLE
Did batch_size influence the final output' quality greatly?

### ISSUE THREAD

**Issue by Danie1Hayes** [otenim/GLCIC-PyTorch#21]
Thanks for your great code!

When i trained my own dataset (roughly 27000 images), it seems phase 3 doesn't improve the sharpness and reality of the inferred holes of output, i.e. ,the  quality is as poor as it was trained in phase 1 (only under the guidance of MSE loss).
Is this because the small batch_size i used? (i set batch_size=8 to fit my poor GPU(GTX1660Ti)) 
I used the newest version of your code(till 2020.10.20), so there isn't any problem with poisson_blend.py ,i guess?

Hope for your response,thanks again for your solid work!

**Comment by otenim** [comment]
@Danie1Hayes 
Batch size influences the quality of final outputs **significantly**.
For example the pretrained model shared in my repo has been trained with batch_size == 96, but when reducing the batch size to 16 or 24, the final outputs get degraded hugely.

Though the proper batch size depends on the data and settings, at least your setting (batch_size == 8) is clearly insufficient😢

**Comment by Danie1Hayes** [comment]
> @Danie1Hayes
> Batch size influences the quality of final outputs **significantly**.
> For example the pretrained model shared in my repo has been trained with batch_size == 96, but when reducing the batch size to 16 or 24, the final outputs get degraded hugely.
> 
> Though the proper batch size depends on the data and settings, at least your setting (batch_size == 8) is clearly insufficient😢

Thanks for your response,i wonder if the batch_size a general **fatal** factor that influence  the inpainting quality in most deep inpainting network, or specific to this paper's work?  Could you share your view on this problem?  
Thanks a lot!

**Comment by otenim** [comment]
@Danie1Hayes 

Basically a too small batch (< 16) size tends to produce not optimal results in most inpainting models, including this paper. I recommend you to try larger batch size (e.g., n >= 64).
